### DISCUSSION TITLE
要快乐

### DISCUSSION THREAD

**Discussion by dongrentianyu** [hxlog/prologue.dev#30] [General]
看了你的博客，不免对社会生活感到悲伤，但你要快乐。就像那句话说的，悲伤是留给读者的，快乐是留给自己的。

**Comment by ghost** [comment]
我认为真正的快乐只有一种，自我实现带来的快乐 = = 每天起床上班路上都会想为何昨天的自己没摆烂，正是接受了冷酷的现实，想到自己被需要着，此刻的努力过程才有所意义，如果生活在乌托邦之类的，真是“生于忧患，死于安乐”了。

你也是，对快乐和幸福的追求才是最重要的。

> 我们必须接受短暂的失望，但不能放弃拥有无限的希望。

与君共勉。
